Add comment to emphasize required filters hidden in GitBook collapsed field

priscaaeby opened the pull request:
Fix-it week https://snyksec.atlassian.net/browse/DP-3105
The required field is hidden when request body in GitBook is collapsed for V1 reporting API. I think we can't change that GitBook behaviour, but we can add a comment to emphasize there is a required field in the filters

priscaaeby committed:
Add comment to emphasize required filters hidden in GitBook collapsed field

mikeromard commented:
I've expanded everything on the reporting API reference page on the [preview](https://docs.snyk.io/~/revisions/8OT8pbepxU4WoUvu2ix6/snyk-api/reference/reporting-api-v1) for this PR, and I don't see the new text in the output. That suggests to me that it needs to be added to a different spot in the spec, though I'm not sure where.

@priscaaeby where in the output do you want/expect this text to appear?

We'll need to change the text itself too, as we try to avoid using "please" in the docs.

priscaaeby committed:
Update description of required filters

priscaaeby commented:
> I've expanded everything on the reporting API reference page on the [preview](https://docs.snyk.io/~/revisions/8OT8pbepxU4WoUvu2ix6/snyk-api/reference/reporting-api-v1) for this PR, and I don't see the new text in the output. That suggests to me that it needs to be added to a different spot in the spec, though I'm not sure where.
> 
> @priscaaeby where in the output do you want/expect this text to appear?
> 
> We'll need to change the text itself too, as we try to avoid using "please" in the docs.

@mikeromard I updated the text of the description. I wonder if it was not working as I've inserted quotes, I fixed that. Can you please retry?

priscaaeby committed:
Try to remove allof

priscaaeby committed:
Remove dash

priscaaeby committed:
Fix

mikeromard commented:
@priscaaeby were you ever able to verify that you're seeing the changes you expected to see on the preview?

mikeromard commented:
Closing this PR due to lack of response.
